.github/workflows/build.yml

```diff
@@ -123,6 +118,54 @@ jobs:
       - name: Build Storybook
         run: yarn storybook:build
 
+  # Test initialization of a new project
+  build_project:
+    name: Initialize and Test Build of Project Codebase
+    runs-on: ubuntu-latest
```

Comment:
Are we going to run out of Github action minutes using their runner here instead of `self-hosted`?

Reply:
I've confirmed that builds from this repo aren't counting against our action minutes allocation.